specification/applicationinsights/resource-manager/Microsoft.Insights/stable/2020-02-02/components_API.json

```diff
@@ -0,0 +1,766 @@
+          "description": "Token used to authenticate communications with between Application Insights and HockeyApp.",
+          "readOnly": true,
+          "externalDocs": {
+            "url": "https://aka.ms/sls76s"
+          }
+        },
+        "provisioningState": {
+          "type": "string",
+          "description": "Current state of this component: whether or not is has been provisioned within the resource group it is defined. Users cannot change this value but are able to read from it. Values will include Succeeded, Deploying, Canceled, and Failed.",
+          "readOnly": true
+        },
+        "SamplingPercentage": {
+          "type": "number",
+          "format": "double",
+          "description": "Percentage of the data produced by the application being monitored that is being sampled for Application Insights telemetry.",
+          "externalDocs": {
+            "url": "https://aka.ms/sl0yqd"
+          }
+        },
+        "ConnectionString": {
+          "type": "string",
+          "readOnly": true,
+          "description": "Application Insights component connection string."
+        },
+        "RetentionInDays": {
+          "type": "integer",
+          "description": "Retention period in days.",
+          "readOnly": true
+        },
+        "DisableIpMasking": {
+          "type": "boolean",
+          "description": "Disable IP masking."
+        },
+        "ImmediatePurgeDataOn30Days": {
+          "type": "boolean",
+          "description": "Purge data immediately after 30 days."
+        },
+        "WorkspaceResourceId": {
+          "type": "string",
+          "description": "Resource Id of the log analytics workspace which the data will be ingested to. This property is required to create an application with this API version. Applications from older versions will not have this property."
+        },
+        "LaMigrationDate": {
+          "type": "string",
+          "readOnly": true,
+          "description": "The date which the component got migrated to LA, in ISO 8601 format.",
+          "format": "date-time"
+        },
+        "PrivateLinkScopedResources": {
+          "readOnly": true,
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/PrivateLinkScopedResource"
+          },
+          "description": "List of linked private link scope resources."
+        },
+        "publicNetworkAccessForIngestion": {
+          "description": "The network access type for accessing Application Insights ingestion.",
+          "$ref": "#/definitions/PublicNetworkAccessType"
+        },
+        "publicNetworkAccessForQuery": {
+          "description": "The network access type for accessing Application Insights query.",
+          "$ref": "#/definitions/PublicNetworkAccessType"
+        },
+        "IngestionMode": {
+          "type": "string",
+          "description": "Indicates the flow of the ingestion.",
+          "default": "LogAnalytics",
```

Comment:
The default value changed from "ApplicationInsights" to "LogAnalytics" is it by design?

Reply:
@akning-ms Do we need breaking change review for Cross Version Breaking Change?